src/ngrx/8-test-api-effects/test-api-effects.md

```diff
@@ -34,9 +34,75 @@ The `--watch` switch will rerun your tests whenever a code file changes. You can
 
 ## Description
 
-TODO: remove or use parts of for an intro for the section
+When testing Effects, we will verify that a specific Action gets dispatched depending on the circumstances when an Effect is triggered. In our case, we are working with Effects that depend on an API response. We can take advantage of spies to simulate different API responses.
 
-You'll need to copy the contents of the test file to run tests for your Effects.
+
+## Update `login.effects.spec.ts`
+
+We will walk through updating `src/app/store/login/login.effects.spec.ts` to run tests for your Effects.
+
+
+### Setting Up our TestBed
+
+When testing Effects, we will need to mock the [`Actions` class](https://ngrx.io/api/effects/Actions) since it plays a major role on [how Effects work](https://ngrx.io/guide/effects#writing-effects). NgRx provides a convenient way to do this through [`provideMockActions()`](https://ngrx.io/api/effects/testing/provideMockActions):
+
+<details open>
+<summary>src/app/store/login/login.effects.spec.ts</summary>
+@diff ../7-create-api-effects/login.effects.spec.ts ./login.effects.spec-setup-test-bed.ts only
+</details>
+
+This will allow us to set `actions$` to be an `Observable` that emits whatever Action we want for our tests:
```

Comment:
"this will allow" is a little confusing to me, how about:
One way to Mock Actions is to set ...

Reply:
why it is confusing: what is `this`?